tests/test_project_management.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for firebase_admin.project_management."""
+
+import json
+
+import pytest
+
+import firebase_admin
+from firebase_admin import project_management
+from tests import testutils
+
+BASE_URL = 'https://firebase.googleapis.com'
+
+TEST_PROJECT_ID = 'hello-world'
```

Comment:
Make the value 'test-project-id' or something similar for clarity.

Reply:
Done.